rke/templates/flannel_v0.14.0-rancher2.go

```diff
@@ -0,0 +1,277 @@
+package templates
+
+const FlannelTemplateV0_14_0Rancher2 = `
```

Comment:
Could add a comment here for Rancher changelog listing the change so we can track the reasoning for a new template? example here: https://github.com/rancher/kontainer-driver-metadata/blob/dev-v2.6/rke/templates/nginx-v1.2.0-rancher2.go#L4

Reply:
thanks for pointing that out. 
added it.